src/parquet/thrift/CMakeLists.txt

```diff
@@ -22,13 +22,29 @@ set(THRIFT_SRCS
 set_source_files_properties(parquet_types.cpp PROPERTIES
   COMPILE_FLAGS -Wno-unused-variable)
 
-add_library(parquet_thrift STATIC
-  ${THRIFT_SRCS}
-)
+if (PARQUET_BUILD_SHARED)
+  add_library(parquet_thrift_shared SHARED
+    ${THRIFT_SRCS}
+  )
```

Comment:
Thanks,
I had the same problem with thrift, so I installed it using the similar fPIC option:
`./boostrap.sh && ./configure --without-php_extension --without-tests --without-qt4 CPPFLAGS=-fPIC`
And it is ok now.